modules/sdk/pages/overview.adoc

```diff
@@ -3,31 +3,38 @@
 [abstract]
 Couchbase offers numerous software development kits (SDKs) that provide programmatic access to Couchbase Server.
 
-xref:2.10@c-sdk::start-using-sdk.adoc[C SDK 2.10] ::
+xref:3.0dp1@c-sdk::start-using-sdk.adoc[C SDK 3.0] ::
```

Comment:
I admire your motivation to version developer previews :)
I would have used 3.0 as the version and have a big WARNING on each page to tell users its in dev preview.